app/src/main/res/layout/story_chapter_view.xml

```diff
@@ -53,6 +53,38 @@
           app:layout_constraintTop_toTopOf="parent"
           app:lessonThumbnail="@{viewModel.chapterThumbnail}" />
 
+        <com.github.mmin18.widget.RealtimeBlurView
```

Comment:
How about if we use Render script?

reference - https://futurestud.io/tutorials/glide-custom-transformation

@rt4914 Any thoughts on this?

Reply:
I looked into the [RealtimeBlurView Library](https://github.com/mmin18/RealtimeBlurView/blob/82df3520b4b377313ea17f27ee54a0410be077ad/library/src-blurimpl/SupportLibraryBlurImpl.java#L21) it uses RenderScript so it would be better rather using the library

Reply:
@anandwana001 I personally prefer non-third-party-library solution. I think we should create a separate PR using this script and see results and based on that finalise things.